content/3_Silver/Intro_Sorted_Sets.mdx

```diff
@@ -126,29 +126,57 @@ which can only be used once before calling the `next()` method.
 <PySection>
 
 In Python, we can use `iter()` to obtain the iterator object of any iterable.
-This returns the first element, and using `next()` on the iterator gives you the
-next one. Below, a dictionary is used instead of a set because dictionaries keep
-order.
+Using `next()` on the iterator lets you iterate through the iterable.
 
 ```py
-nums = {1: None, 7: None, 0: None, 2: None, 4: None}
+nums = {1, "three", 0, (1, 3, 5), 4}
 iterator = iter(nums)
 
-print(iterator)  # <set_iterator object at 0x14765f3eb6c0>
-print(next(iterator))  # 1
-print(next(iterator))  # 7
 print(next(iterator))  # 0
-print(next(iterator))  # 2
+print(next(iterator))  # 1
 print(next(iterator))  # 4
+print(next(iterator))  # three
+print(next(iterator))  # (1, 3, 5)
+
+# Note:
+# The numbers appear ordered when iterating due to the hashing implementation
+# Sets are not ordered, so do not expect this behavior
 ```
 
-<Warning>
+Python's iterators are fundamental to iteration and are used in for loops and
+tuple unpacking, for example. This is useful when you want lower level control
+over your iteration.
 
-As of Python 3.6, dictionaries are ordered by **insertion order**.
-You can also use an `OrderedDict` from `collections`, which is built with the goal of being ordered
-and thus has more features and slightly different speeds compared to a regular Python dictionary.
+In competitive programming however, you will find that the `itertools` library
```

Comment:
wait this module should be about sorted sets, not sure why you decided to include this bit

Reply:
Whoops I went out of scope. I'll take it out.

Surprisingly there is not a lot of usage of `itertools` in Usaco Guide though. It saves a lot of time.

Reply:
no yeah that’s for sure
unfortunately python is just too balls slow